examples/scripts/utils.py

```diff
@@ -18,17 +18,33 @@
 
 import os
 import tarfile
+from glob import glob
 from pathlib import Path
 
 
+def make_custom_plugins(plugins_dir: str) -> list():
+    task_plugins = list()
+    for dirpaths in glob(f"{plugins_dir}/*/"):
+        dirnames = os.path.basename(Path(dirpaths))
+        src_files = glob(f"{plugins_dir}/{dirnames}/*.py")
+        tarball_filename = f"custom-plugins-{dirnames}.tar.gz"
+        make_tar(
+            source_dir=src_files,
+            tarball_filename=tarball_filename,
+            working_dir=plugins_dir,
+        )
+        task_plugins.append(dirnames)
+    return task_plugins
+
+
```

Comment:
I think this function should be in the `register_task_plugins.py` file. It's specific to that script such that it's unlikely that it will need to be used in other scripts.